src/identity/components/GlobalHeader/DefaultEntities.tsx

```diff
@@ -0,0 +1,17 @@
+// Types
+import {UserAccount} from 'src/client/unityRoutes'
+import {OrganizationSummaries} from 'src/client/unityRoutes'
+
+export const emptyAccount: UserAccount = {
```

Comment:
Flagging for a code review re: what the appropriate default state should be here. I think we should set the `setState` default values to an object (or array with one object as applicable) with the same shape, that's empty with everything set to false. 

Using `null` or an empty object is kind of anti-pattern in my opinion, and it causes type issues down the line unless we add conditionals to handle those cases.

Reply:
👍 Also imported the same default empty object into the reducer.